docs/getting-started/composing-components.md

```diff
@@ -5,14 +5,66 @@ title: Composing Components
 
 You can compose components together from other components in your workspace or from components in the cloud.
 
-## Composing from your Local Workspace
+## Create more Components
+
+To create other components you can use `bit create react-component` followed by your component name and if you want it to appear in a specific folder then add the `--namespace` flag followed by the folder name.
+
+```bash
+bit create react-component card --namespace ui
+```
+
+```bash
+the following 1 component(s) were created
+
+my-scope/ui/card at my-scope/ui/card
+    card.composition.tsx
+    card.docs.mdx
+    card.spec.tsx
+    card.tsx
+    index.ts
+```
+
+:::note creating multiple components
+You can create multiple components by adding more component names after the `create react-component` command.
+:::
+
+## Install Dependencies
+
+As we had added a test file that includes dependencies for Testing Library and Chai we will need to install them:
+
+```sh
+bit install @testing-library/react chai
+```
+
+## Composing Component
+
+We will need to modify the Card component as for now all it does is print out a `<p>` tag. Let's change it to have a div and lets import our Button component inside it.
 
 When importing a component into another component you cannot use a relative path. You must use your componentID. In the workspace UI you will see the package name for your component which you can copy to import it.
 
+
 ```js
+import React from 'react';
 import { Button } from '@my-scope/ui.button'
```

Comment:
this is magic for me. how did that happen? i'm used to do relative imports for components, or setting up custom aliases to resolve relative paths... with features/tools like:

* [webpack resolve](https://webpack.js.org/configuration/resolve/)
* [tsconfig resolving](https://www.typescriptlang.org/docs/handbook/module-resolution.html)
* Vue absolute paths
* [babel module resolver](https://github.com/tleunen/babel-plugin-module-resolver)
* `NODE_PATH` environment variable

what bit did so i can do this? is this bit magic? what's the mechanism behind it?

Reply:
emm you tell me

Reply:
For each component in the workspace Bit creates a module in the `node_modules` directory. you must use it if you want to use a bit-component by another component (or even by your app), as that module contains the dist-output of the component.

you can actually see it if you open your `node_modules` directory and open this dir: `@my-scope/ui.button`. it contains symlinks to the component's' source code, auto-generated `package.json` with all dependencies and the component `dist`.

Bit doesn't allow for relative require/import statements, as this couples your component to a specific directory strcuture. using an absolute module name from `node_modules` helps with the de-coupling of components from the workspace and each other.

I think that right after the example you can add a short paragraph called "component module links". in it elaborate on `@my-scope/ui.button` path:

"
Bit creates a module for each component in the workspace. These modules are linked in the `node_modules` directory and contains it's build output and auto-generated `package.json`. To see this in your workspace, browse the `node_modules/@my-scope/ui.button` directory.  
To import a component as a dependency you must use the module link. This way your component is not coupled to a specific directory structure in the workspace, which makes them transferable between workspaces.

> Generating module links
>
> To regenerate module-links for components run the `bit link` command.
"